api/Document.json

```diff
@@ -8146,17 +8146,11 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/onvisibilitychange",
           "support": {
-            "chrome": [
-              {
-                "version_added": "33"
-              },
-              {
-                "version_added": "13",
-                "prefix": "webkit"
-              }
-            ],
+            "chrome": {
+              "version_added": "62"
```

Comment:
Can you add a note for this similar to what's already there for Edge and Safari, and maybe make all the notes consistent?

Also this should be mirrored to Opera, which would look like removal in Opera 15 and then later adding back.

Reply:
I have not been able to confirm that the behavior described in Safari and Edge is also in Chromium, though?

Reply:
The (unprefixed) visibilitychange event was introduced in Chrome 33, so I mean a note similar to this one: `"Before Edge 18, this event handler attribute was not supported; however, the event itself was supported since Edge 12. The event can be listened to via <code>document.addEventListener('visibilitychange', function() {});</code>."`

The note would also apply to Chrome for Android and all other Chromium-based browsers here.